### PR TITLE
Warmup with comprehensive test suite

### PR DESCRIPTION
**Pull request by nc-alex-davis**
- I tried to do some edge cases, but I'm a little unsure about it, ngl

### CODE REVIEW

File: session-outline.md [hunk 1]
```diff
@@ -17,5 +17,5 @@
 - Using iteration, usually in the form of a for…loop, to iterate over a string or array successfully
 - Ability to access elements in an array or characters in a string by their index
 - Accessing properties and values on an object using both dot and square bracket notation
-- Using conditional logic to assert against different conditions, usually in the form of if/else statements, ternaries, or switch statements
+- Using conditional logic to assert against different conditions, usually in the form of if/else statements (perhaps ternaries, or switch statements)
```
Comment (AlanDavison) [inline session-outline.md:20]: Perhaps is a good word. Good addition.
